src/AccessibilityInsights.SharedUxTests/FileIssue/FileIssueActionTests.cs

```diff
@@ -18,9 +18,9 @@ namespace AccessibilityInsights.SharedUxTests.FileIssue
     [TestClass]
     public class FileIssueActionTests
     {
-        static readonly Uri FAKE_SERVER_URL = new Uri("https://myaccount.visualstudio.com/");
-
 #if FAKES_SUPPORTED
+        static readonly string DISPLAY_NAME = "GitHub";
```

Comment:
Uri couldn't be const. string can:

```suggestion
        const string DISPLAY_NAME = "GitHub";
```